tools/content-generator/src/test/java/org/projectnessie/tools/contentgenerator/ITDeleteContent.java

```diff
@@ -53,7 +53,7 @@ void deleteContent() throws NessieNotFoundException {
             CONTENT_KEY.getElements().get(0),
             "--key",
             CONTENT_KEY.getElements().get(1));
-    assertThat(proc.getExitCode()).isEqualTo(0);
```

Comment:
I see this change in several places, but I'm failing to see why we need it.

Reply:
`proc` has a nice `toString()` - so the whole information is printed when it fails